pkg/nodeconfig/nodeconfig.go

```diff
@@ -526,6 +526,8 @@ func generateKubeletConfiguration(clusterDNS string) kubeletconfig.KubeletConfig
 			"SCTPSupport":                    true,
 			"ServiceNodeExclusion":           true,
 			"SupportPodPidsLimit":            true,
+			"CSIMigrationAzureFile":          false,
+			"CSIMigrationvSphere":            false,
```

Comment:
I see this is targeted at release-4.12 so we should be good as long as we have picked up the feature gate unlock in o/k.

Reply:
@aravindhp Yeah this is targeted specifically at 4.12 to match the feature flags there.
I'm not completely sure what the final state of CSI for 4.13  is going to be on the Linux side, but I don't think the in-tree azure-file drivers will be used, as the plugin hit GA in kube 1.26

Reply:
Looking into it with the azure-file flag being GA in 1.26, it cannot be disabled

Reply:
The unlock is only needed for 1.26 and release-4.12 is on 1.25.

Reply:
The migration for vSphere is already false on 4.12 though. So there is no need to disable it explicitly.

Reply:
@jsafrane please chime in here.

Reply:
We handle CSIMigrationvSphere & AzureFile via FeatureGates object. [Default 4.12 OCP FeatureGates disable them](https://github.com/openshift/api/blob/release-4.12/config/v1/types_feature.go#L133-L143), TechPreviewNoUpgrade enables them.

In 4.13, we don't say anything in openshift/api, i.e. both are be enabled today, incl. CSIMigrationvSphere. We will disable its behavior in a hackish way using a carry patch + env. var. https://github.com/openshift/kubernetes/pull/1514

Reply:
@sebsoto looks like you don't have to explicitly disable this here. Or did you discover something else otherwise in your tests? It is possible that the change in the OpenShift API is not reflected in Windows nodes.

Reply:
> looks like you don't have to explicitly disable this here

I don't see how thats the case, Windows nodes dont use the FeatureGates defined there, we use our own set. The Linux worker node feature gates are defined in the worker MachineConfig, which we don't use.